
{'date': 'Mon Nov 25 2019 01:52:56.364 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 14
  introd = Cena (img = "https://i.imgur.com/Bcnfg0C.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 25 2019 01:54:13.390 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  def quarto():
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 25 2019 01:54:26.962 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 22
  travesseiro = Elemento(FOCO, x=90, y=180, w=50, h=50, cena=quartotalita, style={"opacity": 0.0}, vai=textotravesseiro.vai)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 01:54:35.779 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  colete = Elemento (img = linkcolete,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 01:55:47.133 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 26
  colete = Elemento (img = linkcolete,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 01:56:05.894 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 30
  protetorsolar = Elemento (img = linkprotetorsolar,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 02:00:24.803 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 15
  def quarto(self):
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 25 2019 02:00:39.512 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 16
  self.introd = Cena (img = "https://i.imgur.com/Bcnfg0C.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 25 2019 02:00:46.7 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 17
  self.quartotalita = Cena (img = "https://lh5.googleusercontent.com/-fs1hatHWU9s/UUpy0DJqlXI/AAAAAAAAbs4/Vy1LL28sPeY/s400/tumblr_lt4n2aSjsX1qmvaoo.gif")
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 02:00:52.162 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 19
  talita = Elemento (img = linkdatalita, 
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 02:01:02.22 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 28
  colete = Elemento (img = linkcolete,
  ^
IndentationError: unexpected indent
'''},
{'date': 'Mon Nov 25 2019 02:04:25.753 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    Jogo()
  module <module> line 23
    		textotravesseiro = Texto (quartotalita, "Não era bem isso que eu estava procurando..")  
NameError: name 'quartotalita' is not defined
'''},
{'date': 'Mon Nov 25 2019 02:04:53.991 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    Jogo()
  module <module> line 23
    		textotravesseiro = Texto (self.quartotalita, "Não era bem isso que eu estava procurando..")  
AttributeError: 'Jogo' object has no attribute 'quartotalita'
'''},
{'date': 'Mon Nov 25 2019 02:05:32.160 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    Jogo()
  module <module> line 23
    		textotravesseiro = Texto (self.quartotalita, "Não era bem isso que eu estava procurando..")  
AttributeError: 'Jogo' object has no attribute 'quartotalita'
'''},
{'date': 'Mon Nov 25 2019 02:42:54.80 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 41
    Jogo()
  module <module> line 23
    		textotravesseiro = Texto (self.quartotalita, "Não era bem isso que eu estava procurando..")  
AttributeError: 'Jogo' object has no attribute 'quartotalita'
'''},
{'date': 'Mon Nov 25 2019 02:45:10.109 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    Jogo()
  module <module> line 17
    			self.introd.vai=quartotalita.vai()
NameError: name 'quartotalita' is not defined
'''},
{'date': 'Mon Nov 25 2019 02:45:22.32 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 29
    Jogo()
  module <module> line 17
    			self.introd.vai=quarto.vai()
AttributeError: 'quarto' object has no attribute 'vai'
'''},
{'date': 'Mon Nov 25 2019 02:46:23.89 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''
 module <string> line 28
  self.quartotalita.vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Mon Nov 25 2019 02:46:30.854 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    Jogo()
  module <module> line 17
    			self.introd.vai=quarto.vai()
TypeError: vai() missing 1 positional argument: self
'''},
{'date': 'Mon Nov 25 2019 02:46:46.341 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 30
    Jogo()
  module <module> line 17
    			self.introd.vai=self.quarto.vai()
AttributeError: 'Jogo' object has no attribute 'quarto'
'''},
{'date': 'Mon Nov 25 2019 02:48:08.685 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    quarto.inicia()
TypeError: inicia() missing 1 positional argument: self
'''},
{'date': 'Mon Nov 25 2019 02:48:52.935 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 32
    self.quarto.inicia()
AttributeError: 'GUI' object has no attribute 'quarto'
'''},
{'date': 'Mon Nov 25 2019 02:49:17.835 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 31
    Jogo()
  module <module> line 17
    			self.introd.vai=self.quarto.vai()
AttributeError: 'Jogo' object has no attribute 'quarto'
'''},
{'date': 'Mon Nov 25 2019 02:56:27.43 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    Jogo()
  module <module> line 24
    			self.botao.vai=quarto.vai()
TypeError: vai() missing 1 positional argument: self
'''},
{'date': 'Mon Nov 25 2019 02:56:36.671 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    Jogo()
  module <module> line 24
    			self.botao.vai=quarto.vai()
TypeError: vai() missing 1 positional argument: self
'''},
{'date': 'Mon Nov 25 2019 03:02:56.582 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 38
    Jogo() 
  module <module> line 21
    			self.botao.entra(introd)
NameError: name 'introd' is not defined
'''},
{'date': 'Mon Nov 25 2019 03:37:08.828 GMt-1000 (Horário Padrão do Havaí e Ilhas Aleutas) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 34
    self.talita.entra(self.introd)
AttributeError: 'GUI' object has no attribute 'talita'
'''},